Implement generic parser for the streaming call data (Audio, Transcription or Live captions)

abhishesingh-msft opened the pull request:
Implement generic parser for the call data (Audio, Transcription or LiveCaption) streaming over websocket

1> Extend Audio & Transcription websocket models from common model StreamingData.
2> StreamingDataParser is now a generic parser for all the call data streaming for Audio, Transcription or any other future addition such as Live Captions.
4> Regenerate model & create api-view.
5> Refactor Audio & Transcription model test cases.

abhishesingh-msft committed:
Implement generic parser for the call data (Audio, Transcription or LiveCaption) streaming over websocket

1> Extend Audio & Transcription websocket models from common model StreamingData.
2> StreamingDataParser is now a generic parser for all the call data streaming for Audio, Transcription or any other future addition such as Live Captions.
4> Regenerate model & create api-view.
5> Refactor Audio & Transcription model test cases.

abhishesingh-msft committed:
Implement generic parser for the call data (Audio, Transcription or LiveCaption) streaming over websocket

1> Extend Audio & Transcription websocket models from common model StreamingData.
2> StreamingDataParser is now a generic parser for all the call data streaming for Audio, Transcription or any other future addition such as Live Captions.
4> Regenerate model & create api-view.
5> Refactor Audio & Transcription model test cases.

abhishesingh-msft committed:
Implement generic parser for the call data (Audio, Transcription or LiveCaption) streaming over websocket

1> Extend Audio & Transcription websocket models from common model StreamingData.
2> StreamingDataParser is now a generic parser for all the call data streaming for Audio, Transcription or any other future addition such as Live Captions.
4> Regenerate model & create api-view.
5> Refactor Audio & Transcription model test cases.

abhishesingh-msft committed:
Fixing unit tests

abhishesingh-msft committed:
Fixing unit tests

abhishesingh-msft committed:
Fixing unit tests

azure-sdk commented:
**API change check**

APIView has identified API level changes in this PR and created following API reviews.

[Azure.Communication.CallAutomation](https://apiview.dev/Assemblies/Review/6f99e13b3e5d4d468576843e06835d20)

amnguye committed:
Renamed Storage PublicAudience to DefaultAudience, Get..Audience to Create..Audience (#39083)

azure-sdk committed:
Increment package version after release of Azure.Monitor.OpenTelemetry.AspNetCore (#39129)

sofiar-msft committed:
Add livetests to PPE (#39110)

* Update tests.yml

* Skip SIP tests on int and ppe

* Skip SIP tests on ppe

* Add flag to skip sip configuration tests

* Add flag to skip sip configuration in matrix

* Update to net462

* Remove Siprounting flag

* Remove Siprounting flag from matrix

* Add Siprounting flag to matrix

azure-sdk committed:
Detect all file diff types for eng/common changes (#39128)

Fixes https://github.com/Azure/azure-sdk-tools/issues/5882

We need to set the difffilter to empty instead of the default of exclude deleted files when we are trying to verify there are no changes under eng/common.

See test PR https://github.com/Azure/azure-sdk-for-python/pull/32348 which demonstrates us not detecting a deleted file under eng/common. I'll use that same test PR to verify this now catches that issue.

Co-authored-by: Wes Haggard <weshaggard@users.noreply.github.com>

cochi2 committed:
Adding CorrelationId to TranscriptionMetadata (#39130)

mikehang-msft committed:
Update ChangeLog file (#39125)

jalauzon-msft committed:
[Storage][DataMovement] Update checkpointer to read/write to job file - Part 2 (#39101)

ArthurMa1978 committed:
Release Azure.ResourceManager.ContainerServiceFleet (#39106)

* Release Azure.ResourceManager.ContainerServiceFleet

* update

azure-sdk committed:
Increment package version after release of Azure.ResourceManager.ContainerServiceFleet (#39139)

ArcturusZhang committed:
fix pageable operation issue in purview administration library (#39009)

live1206 committed:
use directive to set api-version as client for Resources_GetById (#39141)

azure-sdk committed:
Update AutoRest C# version to 3.0.0-beta.20231008.1 (#39143)

* Update Generator Version 3.0.0-beta.20231008.1

* Update SDK codes

* Update SDK codes

* Update SDK codes

* Update SDK codes

* Update SDK codes

* Update SDK codes

azure-sdk committed:
Update Generator Version 3.0.0-beta.20231008.2 (#39148)

zackliu committed:
[WebPubSubClient] Parsing message returns IList (#39137)

* Parsing message returns IList

* update api

* Use IReadonlyList

nisha-bhatia committed:
Update CHANGELOG.md (#39160)

azure-sdk committed:
Increment package version after release of Azure.Search.Documents (#39161)

nisha-bhatia committed:
[Azure.Monitor.Inestion] Update CHANGELOG.md for GA Release (#39165)

wiboris committed:
Adding Support to override the ClientRequestID guid format for test recording sessions. (#39133)

* adding an enviroment variable "RECORDING_DEFAULT_ClIENT_GUID" to support using the default string format for ClientID's in test recording mode

* changing code to support storing the enviroment variable into the test recordings variables

* handling the case if the variable is not present in the recording

* changing from enviroment var to property of RecordedTestBase

* removing files

* whitespace fix

* update

* recovered deleted file

* restored deleted code

* fix

* feedback

(cherry picked from commit 240efa7ad801712117eb3624791431f92aa132d6)

* removed dead code

* whitespace

live1206 committed:
Move ResourcesRestOperations to generated (#39150)

* Move most of ResourcesRestOperations to generated

* make api-version as method parameter for Resources_GetById

* Add comments

azure-sdk committed:
Increment package version after release of Azure.Data.AppConfiguration (#39169)

azure-sdk committed:
Increment package version after release of Azure.Monitor.Ingestion (#39168)

azure-sdk committed:
Update Generator Version 3.0.0-beta.20231009.1 (#39173)

glorialimicrosoft committed:
Add azure communication messages sdk (#39167)

* Add Messages SDK

* Work around the issue with nextLink never empty, add more tests

* Update CHANGELOG.md

* Update autorest and use Github link for input file

* Update README.md

* dropping the Value suffix on everything but the base class (MessageTemplateValue)

* change the param name from url to uri, filename to fileName

* Update CHANGELOG.md

* revert accidental check in of RecordedTestMode

* sanitize session records

* remove unneeded import

* update to 2023-08-24-preview api version

* update CommunicationMessagesClientOptions api version to 2023-08-24-preview

* re-run live test recording

* fix broken link

* use latest swagger

* Remove unsupported auth type, add more live tests, and re-run the live tests

* use the swagger that does not have operation-id response header

* Take Value suffix off MessageTemplateQuickAction

* update sdk to latest

- change autorest to using readme instead of swagger input-file
- update Butotn from IDictionary to IEnumerable for maintaining orders
- autorest point to azure-rest-api-specs-pr
- re-run live tests to get new session records

* update the source file to the public azure specs repo

* fix recording

* run "eng\scripts\Export-API.ps1 communication"

* run "eng\scripts\CodeChecks.ps1 -ServiceDirectory communication"

JoshLove-msft committed:
Prepare for release (#39184)

scottaddie committed:
Replace Azure.Identity AAD references (#39123)

jalauzon-msft committed:
[Storage][DataMovement] Add ProviderId to resources and job plan file (#39163)

azure-sdk committed:
Update github-event-processor version to 1.0.0-dev.20231010.2 (#39189)

Co-authored-by: James Suplizio <jasupliz@microsoft.com>

vivekmore-msft committed:
Fixed tansxription review comments (#39195)

azure-sdk committed:
Increment package version after release of Azure.Communication.Messages (#39185)

JoshLove-msft committed:
Fix broken link (#39188)

JoshLove-msft committed:
Prepare for release (#39203)

azure-sdk committed:
Increment package version after release of Microsoft.Azure.WebJobs.Extensions.EventHubs (#39201)

JoshLove-msft committed:
Add new system events (#39202)

* Add new system events

* Fix source generator

* refactor

JoshLove-msft committed:
Prepare for release (#39205)

azure-sdk committed:
Use url version spec for autorest.* packages (#39131)

Co-authored-by: Patrick Hallisey <pahallis@microsoft.com>

azure-sdk committed:
Increment package version after release of Azure.Messaging.ServiceBus (#39204)

mcgallan committed:
Fix OperationalInsights Type Issue (#38425)

* update

* update

m-redding committed:
[Core] Update RequestFailedException to use source generation for serialization (#38996)

* iteration 1

* iteration 1

* tweak logic

* simplification

* adjust comments

* Update sdk/core/Azure.Core/src/RequestFailedException.cs

Co-authored-by: Eric Erhardt <eric.erhardt@microsoft.com>

* feedback fixes

---------

Co-authored-by: Eric Erhardt <eric.erhardt@microsoft.com>

JoshLove-msft committed:
Rename property in ResourceNotificationsResourceUpdatedDetails (#39210)

azure-sdk committed:
Use better description for single repo sparse checkout (#39209)

Co-authored-by: Patrick Hallisey <pahallis@microsoft.com>

scottaddie committed:
Document sovereign cloud support in LogsIngestionClient (#39221)

JoshLove-msft committed:
Dispose clients on host shutdown rather than listener dispose (#39225)

* Dispose clients on host shutdown rather than listener dispose

* Remove project reference

* Update version - for some reason the auto-update job failed

azure-sdk committed:
Update Generator Version 3.0.0-beta.20231012.5 (#39239)

amariwest-msft committed:
Add PauseOnStart Recording Option  (#39132)

* generated files using swagger

* removed extra swagger file

* modified unit tests

* undo status code change for mute participant in generated files

* merged with main

* merged with main

* reverted 202 change

* Both status code changed to 202

* modified test cases to check for 200 status code

* regenerated code

* regenerated code

* Revert "regenerated code"

This reverts commit a5a9e0e5b8296fbf90a49dc8ebacd62f75298690.

* Added TransciptionDataRequest Change

* generated from latest swagger

* generated netstandard2.0.cs

* edited changelog

* Revert "generated netstandard2.0.cs"

This reverts commit 2a39b2d865a5a223e0edaf7e3615dc16562e85b7.

* Revert "edited changelog"

This reverts commit 7eb14250f0a123983d82e606ed8c7fb5d4585a86.

* added line to changelog and standard2.0.cs

* Update CHANGELOG.md

audunn committed:
[NetAppFiles] Update to api-version 2023-05-01 (#38882)

* Update to api-version 2023-05-01

* SystemData

* Sync + systemdata

* update assets

* update assets

* s2ync

* EB.Never

* EB.Never missing using ??

* export api

* change changelog

* assets

* assets

* assets

abhishesingh-msft committed:
Resolve conflicts

abhishesingh-msft committed:
Merge branch 'abhishesingh/generic-streaming-parser' of https://github.com/Azure/azure-sdk-for-net into abhishesingh/generic-streaming-parser
